### PR TITLE
fix(ci): remove tmp fix on deploy script

### PR DESCRIPTION
**Pull request by schoren**
This PR removes a tmp fix on the deploy script. The helm chart is now corrected, so we can fully rely on it

### CODE REVIEW

File: server/http/mappings/data_store.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func (m OpenAPI) DataStoreType(in model.DataStoreType) openapi.SupportedDataStor
 	dsd, exists := dataStoreTypesMapping[in]
 	if !exists {
 		// this should only happen during development,
-		// so it's more an alert for devs than actual error handling``
```
Comment (schoren) [inline server/http/mappings/data_store.go:28]: just a typo in a comment

File: server/http/mappings/data_store.go [hunk 2]
```diff
@@ -94,7 +94,7 @@ func (m Model) DataStoreType(in openapi.SupportedDataStores) model.DataStoreType
 	}
 
 	// this should only happen during development,
-	// so it's more an alert for devs than actual error handling``
```
Comment (schoren) [inline server/http/mappings/data_store.go:97]: just a typo in a comment

File: k8s/deploy.sh [hunk 3]
```diff
@@ -51,19 +49,6 @@ kubectl --namespace $NAME create configmap $NAME --from-file=$CONFIG_FILE --from
...
-  $NAME \
-  --namespace $NAME \
-  --type='json' \
-  -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/args", "value": [
-  "--config",
-  "/app/config/config.yaml",
-  "--provisioning-file",
-  "/app/config/provisioning.yaml"
-]}]'
-
```
Comment (schoren) [inline k8s/deploy.sh:66]: All of this is not happening on the helm chart

File: k8s/deploy.sh [hunk 4]
```diff
@@ -35,12 +35,10 @@ fi
 
 helm repo add kubeshop https://kubeshop.github.io/helm-charts
 helm repo update
-helm upgrade --install $NAME kubeshop/tracetest \
+helm upgrade --install tracetest kubeshop/tracetest \
   --namespace $NAME --create-namespace \
   --set image.tag=$TAG \
-  --set image.pullPolicy=Always \
-  --set service.port=11633 \
-  --set server.httpPort=11633 \
```
Comment (schoren) [inline k8s/deploy.sh:43]: This was also needed from when we changed the default port, which has been the default in the chart for a while

**Review by xoscar — APPROVED** [review]

**Pull request merged**